languages/go/example/example.go

```diff
@@ -16,14 +16,15 @@ func main() {
 	bitwardenClient, _ := sdk.NewBitwardenClient(&apiURL, &identityURL)
 
 	accessToken := os.Getenv("ACCESS_TOKEN")
+	statePath := os.Getenv("STATE_PATH")
```

Comment:
We should mention that this is optional, like we do with the API and identity URLs

Reply:
Updated!